src/simsoptpp/simdhelpers.h

```diff
@@ -129,6 +129,7 @@ using AlignedPaddedVec = std::vector<double, AlignedPaddedAllocator<double>>;
 #endif
 
 #if defined(USE_XSIMD)
+/*
 #if __AVX512F__ 
```

Comment:
If AVX-512 is disabled, shouldn't `__AVX512F__` not be defined, and this section of code would be avoided?